packages/web-react/src/components/Tabs/__tests__/TabPane.test.tsx

```diff
@@ -15,27 +15,25 @@ describe('TabPane', () => {
     (props) => (
       <Tabs selectedTab={1} toggle={() => {}}>
         <TabContent>
-          <TabPane tabId={1} data-testid="TabPaneTestId" {...props} />
+          <TabPane id={1} data-testid="TabPaneTestId" {...props} />
         </TabContent>
       </Tabs>
     ),
     'TabPaneTestId',
   );
 
   classNamePrefixProviderTest(
-    withTabsContext((props) => <TabPane {...props} tabId="test" />, { selectedTabId: 'test' } as TabsContextType),
+    withTabsContext((props) => <TabPane {...props} id="test" />, { selectedId: 'test' } as TabsContextType),
     'TabsPane',
   );
 
   restPropsTest(
-    withTabsContext((props) => <TabPane {...props} tabId="test" />, { selectedTabId: 'test' } as TabsContextType),
+    withTabsContext((props) => <TabPane {...props} id="test" />, { selectedId: 'test' } as TabsContextType),
     'div',
   );
 
   it('should not render tab pane if tab is not selected', () => {
```

Comment:
```
  it('should not render tab pane if tab is not selected', () => {
    render(withTabsContext(TabPane, { selectedId: 'another-tab', selectTab: jest.fn() })({ id: 'test' }));

    expect(screen.queryByRole('tabpanel')).not.toBeInTheDocument();
  });

  it('should render tab pane if tab is selected', () => {
    render(withTabsContext(TabPane, { selectedId: 'test', selectTab: jest.fn() })({ id: 'test' }));
    screen.debug();
    const tabPane = screen.queryByRole('tabpanel');

    expect(tabPane).toBeInTheDocument();
    expect(tabPane).toHaveAttribute('id', 'test');
    expect(tabPane).toHaveAttribute('aria-labelledby', 'test-tab');
    expect(tabPane).toHaveClass('TabsPane');
    expect(tabPane).toHaveClass('is-selected');
  });
```

Reply:
Can you please provide some context for using `queryByRole` or `getByRole`? Is it considered a best practice or is it efficient? I like those suggestions. Just wonder why is better to use it this way. Thanks.

Reply:
I would like to discuss this topic with you or possibly make a presentation to unify our approach on how to use it 😉https://testing-library.com/docs/queries/about#priority

Reply:
> Can you please provide some context for using queryByRole or getByRole? Is it considered a best practice or is it efficient? I like those suggestions. Just wonder why is better to use it this way.

AFAIK:

- `getByRole` fails ➡️ test fails
- `queryByRole` fails ➡️ test is skipped

Reply:
@curdaj Thank you for keeping the eye on the tests! I think we can test things that should **not** be there more often.

Reply:
Ok, thanks for enlightening :-) Let's talk about it and use it.